lib/client/signalflow/request_manager.js

```diff
@@ -390,11 +390,34 @@ function RequestManager(options) {
     //destroy all open SSE connections
   }
 
+  function registerLiveTail(callbacks, overrideRequestId ){
```

Comment:
I would expect you to need a `params` argument here to carry your live tail query.